src/Stripe.net/Services/Accounts/AccountSettingsOptions.cs

```diff
@@ -10,9 +10,6 @@ public class AccountSettingsOptions : INestedOptions
         [JsonProperty("card_payments")]
         public AccountSettingsCardPaymentsOptions CardPayments { get; set; }
 
-        [JsonProperty("dashboard")]
-        public AccountSettingsDashboardOptions Dashboard { get; set; }
-
```

Comment:
you forgot to remove that class from the library right?

Reply:
Yep. I forgot that one. great catch! ptal